cpp/tensorrt_llm/kernels/cutlass_kernels/moe_gemm/moe_kernels.cu

```diff
@@ -1332,6 +1329,10 @@ __global__ void computeStridesTmaWarpSpecializedKernel(int64_t const* expert_fir
         reinterpret_cast<TmaWarpSpecializedGroupedGemmInput::INT4GroupwiseParams::SFA const*>(
             quant_params.groupwise.fc2.weight_scales),
         bias2, gemm2_output, router_scales, permuted_row_to_unpermuted_row, expert);
+
+#if (defined(__CUDA_ARCH__) && (__CUDA_ARCH__ >= 900))
+    asm volatile("griddepcontrol.launch_dependents;");
```

Comment:
Is there any perf difference from this overlap, we could split it so GEMM1 is done before and GEMM2 is setup afterwards. I dont think we expect a big speedup though, this is a very small kernel